ui/admin/config/optional-features.json

```diff
@@ -2,5 +2,6 @@
   "application-template-wrapper": false,
   "default-async-observers": true,
   "jquery-integration": false,
-  "template-only-glimmer-components": true
+  "template-only-glimmer-components": true,
+  "no-implicit-route-model": true
```

Comment:
learning: We need this because we previously relied on the fact that if you don't define a model hook for a route, it will use the model from the parent route correct?